sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -39,6 +39,9 @@ import org.apache.spark.sql.catalyst.plans.logical._
 import org.apache.spark.sql.catalyst.rules._
 import org.apache.spark.sql.catalyst.streaming.StreamingRelationV2
 import org.apache.spark.sql.catalyst.trees.TreeNodeRef
+import org.apache.spark.sql.catalyst.trees.TreePattern.{
```

Comment:
Does this pass scalastyle?

Reply:
It seemed ok -- I didn't see scalastyle complained anything. 
But let me know if there's a convention in the Spark codebase.

Reply:
Normally we put it all on a single line. If it passes then I am happy.

Reply:
It passes..